examples/plot_04_tracking_items.py

```diff
@@ -0,0 +1,160 @@
+"""
+.. _example_historization_items:
+
+==================================
+Tracking items using their history
+==================================
+
+This example illustrates how skore can be used to track some items using their history,
+for example tracking some ML metrics over time.
+"""
+
+# %%
+import subprocess
+import time
+
+import matplotlib.image as mpimg
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import plotly.express as px
+from sklearn import datasets
+from sklearn.linear_model import Lasso
+
+import skore
+
+# %%
+# Creating and loading the skore project
+# ======================================
+
+# %%
+
+# remove the skore project if it already exists
+subprocess.run("rm -rf my_project_track.skore".split())
+
+# create the skore project
+subprocess.run("python3 -m skore create my_project_track".split())
```

Comment:
Please can you use tmp directory, as in other examples?

Reply:
Done